server/website/website/parser/base.py

```diff
@@ -157,9 +174,21 @@ def convert_dbms_metrics(self, metrics, observation_time):
             else:
                 raise Exception(
                     'Unknown metric type for {}: {}'.format(name, metadata.metric_type))
-        if self.transactions_counter not in metric_data:
-            raise Exception("Cannot compute throughput (no objective function)")
+        # if self.transactions_counter not in metric_data and self.latency_timer not in metric_data:
+        #     raise Exception("Cannot compute throughput or latency (no objective function)")
+
+        if target_objective is not None and self.target_metric(target_objective) not in metric_data:
+            raise Exception("Cannot find objective function")
+
+        # if target_objective != None:
+        #     metric_data[target_objective] = metric_data[self.target_metric(target_objective)]
+        # else:
+        #     # default
+        #     metric_data['throughput_txn_per_sec'] =  \
+        #       metric_data[self.target_metric(target_objective)]
+
         metric_data['throughput_txn_per_sec'] = metric_data[self.transactions_counter]
+        metric_data['99th_lat_ms'] = metric_data[self.latency_timer]
```

Comment:
I think we only want to show the selected target objective for now, instead of both of them .   Otherwise it will break since we do not have the latency metrics now.

Reply:
Adding more metrics to default metrics will cause the side bar to have the metrics that are not the target objective. For example, we choose throughput as the target objective but in the side bar latency will also present. If we select latency, the website will not break but the plot will never be loaded since there is no such metrics in metric_data. If it is okay, I will only include the selected target objective.

Reply:
We should remove latency from the target objective options since it's not supported (including from the create session view where the user selects the target objective). For now, the only option in target objective should be the throughput.